src/scala/META-INF/native-image/org/pantsbuild/zinc/compiler/README.md

```diff
@@ -0,0 +1,8 @@
+native-image `META-INF/`
+===========================
+
+https://www.graalvm.org/docs/reference-manual/aot-compilation/
+
+https://github.com/oracle/graal/blob/master/substratevm/CONFIGURE.md
```

Comment:
Unless these instructions were used verbatim, it would be good to include a bit more information about how the files were created/generated. And it they were used verbatim, a comment saying so would make this less intimidating!

Reply:
Fixed!